cartographer_ros/configuration_files/backpack_2d.lua

```diff
@@ -32,6 +32,9 @@ options = {
   submap_publish_period_sec = 0.3,
   pose_publish_period_sec = 5e-3,
   trajectory_publish_period_sec = 30e-3,
+  rangefinder_sampling_ratio = 1.,
```

Comment:
Keep in mind that you will have to update all other configuration files (for example, in repos `cartographer_turtlebot`, `cartographer_toyota_hsr`, etc.). This was one reason why I was reluctant to introduce a new option to cartographer_ros. 

Perhaps it might make sense to introduce a common .lua file in cartographer_ros which would have some default settings, and which would be included from other robot-specific .lua files. This way, when a new option is introduced, a default value could be provided in the common settings file, without having to update all other configurations.

Reply:
@cschuet Please make sure this is handled when you merge.